sw/airborne/firmwares/rotorcraft/navigation.c

```diff
@@ -148,11 +148,16 @@ static void send_wp_moved(struct transport_tx *trans, struct link_device *dev)
   static uint8_t i;
   i++;
   if (i >= nb_waypoint) { i = 0; }
+  struct EnuCoor_i wp_i;
```

Comment:
I understand why you have that, but I don't think it should appear in master. Would be nice if you remove it for now, and eventually add it back if you decide to make a pull request for the code that is actually using this.